doc/source/whatsnew/v2.1.0.rst

```diff
@@ -624,11 +624,13 @@ Other
 - Bug in :meth:`DataFrame.reindex` with a ``fill_value`` that should be inferred with a :class:`ExtensionDtype` incorrectly inferring ``object`` dtype (:issue:`52586`)
 - Bug in :meth:`DataFrame.shift` and :meth:`Series.shift` and :meth:`DataFrameGroupBy.shift` when passing both "freq" and "fill_value" silently ignoring "fill_value" instead of raising ``ValueError`` (:issue:`53832`)
 - Bug in :meth:`DataFrame.shift` with ``axis=1`` on a :class:`DataFrame` with a single :class:`ExtensionDtype` column giving incorrect results (:issue:`53832`)
+- Bug in :meth:`Index.sort_values` whena  ``key`` is passed (:issue:`52764`)
```

Comment:
Nit: when a

Reply:
Also - move to the Indexing section?

Reply:
Good catch, will update.

As to the section, its pretty subjective but I usually think of "indexing" as about Series/DataFrame/loc.iloc getitem/setitem methods

Reply:
updated+green